page-templates/evergreen.php

```diff
@@ -26,6 +26,11 @@
 $page_meta_data = get_post_meta( $post->ID );
 $page_meta_data = array_map( 'reset', $page_meta_data );
 
+$redirect_tag_id = get_post_meta( $post->ID, 'redirect_tag_page', true );
```

Comment:
I'm still checking if there's a function for this. In the end we want it to execute the following query:
```sql
select * from wp_termmeta where meta_key='redirect_page' and meta_value={{post_id}}